customfiles/14_0_INSTALLERCONFIG.sample

```diff
@@ -167,9 +165,20 @@ cat >>/usr/local/etc/rc.d/firstboot_depenguin<<"EOF"
 
 name="firstboot_depenguin"
 rcvar=firstboot_depenguin_enable
-start_cmd="firstboot_depenguin_run | logger -s -t 'depenguin'"
+start_cmd="check_for_realtek | firstboot_depenguin_run | logger -s -t 'depenguin'"
```

Comment:
Piping this isn't a good idea (connecting check_for_realtek's standard output to firstboot_depenguin's stdin might cause broken pipe etc.). Best to call `check_for_realtek` from inside `firstboot_depenguin_run`.